applications/launchpad/backend/src/api/mod.rs

```diff
@@ -53,6 +53,9 @@ pub struct ImageInfo {
     image_name: String,
     display_name: String,
     docker_image: String,
+    is_updated: bool,
```

Comment:
```suggestion
    latest: bool,
```
on frontend we assumed
```
export type DockerImage = {
  imageName: string
  displayName: string
  dockerImage: string
  latest: boolean
  pending?: boolean
  error?: string
  progress?: number
  status?: DockerImagePullStatus
}
export enum DockerImagePullStatus {
  Waiting = 'waiting',
  Pulling = 'pulling',
  Ready = 'ready',
}
```

Reply:
The changes are implemented.